### PR TITLE
Add insert GIF shortcut

### PR DESCRIPTION
**Pull request by jdsutherland**
Add a shortcut to launch GIF search <kbd>⌘</kbd><kbd>G</kbd>.

I wanted to additionally add shortcuts to cycle through the gifs but it seems it would require `electron-localshortcut` and might be overly clunky.

_Edit_: actually, I realized <kbd>tab</kbd>  and <kbd>shift</kbd><kbd>tab</kbd> works to cycle through the gifs, so one doesn't have to reach for the mouse after all.

image

image

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add it to https://github.com/sindresorhus/caprine#keyboard-shortcuts

**Comment by jdsutherland** [comment]
@sindresorhus sure

**Comment by veniversum** [comment]
Needs to be added to the menus for other OS (Windows & Linux) as well.

**Comment by jdsutherland** [comment]
@veniversum Oops - thanks for catching that. I've updated it.

File: menu.js [hunk 1]
```diff
@@ -265,6 +265,13 @@ const macosTpl = [
 				}
 			},
 			{
+				label: 'Insert GIF',
+				accelerator: 'CmdOrCtrl+G',
```
Comment (sindresorhus) [inline menu.js]: `CmdOrCtrl+G` => `Cmd+G`

**Comment by sindresorhus** [comment]
Thank you @jdsutherland :)

**Pull request merged**